src/modules/to.test.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import arrTasks from './to_class.js';
+import addTaskItem from './addTaskItem.js';
+import removeTaskItem from './removeTaskItem.js';
+import saveLocal from './saveLocal.js';
+
+document.body.innerHTML = '<ul class="ul-tasks" id="ul-tasks"></ul>';
+arrTasks.tasks = [];
+
+describe('test addTaskItem and LocalStorage', () => {
+  test('add task to show arrTask.tasks length 1', () => {
+    addTaskItem('to check test');
+    expect(arrTasks.tasks).toHaveLength(1);
+  });
+
+  test('test for local storage', () => {
+    saveLocal();
+    expect(JSON.parse(localStorage.getItem('arrTaskLocal'))).toHaveLength(1);
+  });
+
+  test('add task to show arrTask.tasks length 2', () => {
+    addTaskItem('to check test2');
+    expect(arrTasks.tasks).toHaveLength(2);
+  });
+
+  test('test for local storage', () => {
+    saveLocal();
+    expect(JSON.parse(localStorage.getItem('arrTaskLocal'))).toHaveLength(2);
+  });
+});
+
+describe('test removeTaskItem and LocalStorage', () => {
+  test('remove task to show arrTask.tasks length 0', () => {
+    removeTaskItem(1);
+    expect(arrTasks.tasks).toHaveLength(1);
+  });
+
+  test('test for local storage', () => {
+    saveLocal();
+    expect(JSON.parse(localStorage.getItem('arrTaskLocal'))).toHaveLength(1);
+  });
+  test('remove task to show arrTask.tasks length 0', () => {
+    removeTaskItem(1);
+    expect(arrTasks.tasks).toHaveLength(0);
+  });
+
+  test('test for local storage', () => {
+    saveLocal();
+    expect(JSON.parse(localStorage.getItem('arrTaskLocal'))).toHaveLength(0);
+  });
+});
```

Comment:
- [ ] Great job adding tests for adding and removing tasks. However, I cannot see any library set up in your package.json that will run the tests. This is what you have presently in that file.

![test](https://github.com/HenaryKhuraijam/ToDoList/assets/26597432/77d221b4-9367-4eec-81ac-4f753a729302)

Kindly look into it and fix.
*Note:* I reached out to both of you on slack, but there was no response from none of you.

Reply:
Sorry @Qoosim  for not replying in time due to a different time zone. I rechecked the code and library as per your suggestion and fixed the problems. 
Thank you so much for finding out our mistakes.